Source/DayView.swift

```diff
@@ -129,7 +131,9 @@ extension DayView: PagingScrollViewDelegate {
   func scrollviewDidScrollToViewAtIndex(_ index: Int) {
     let timeline = timelinePager.reusableViews[index].timeline
     currentDate = timeline!.date
+    delegate?.dayView(dayView: self, willMoveTo: currentDate)
```

Comment:
I think, this method should be invoked before scrolling occurs. But I can fix it later.